storage/file.go

```diff
@@ -425,6 +425,10 @@ func (f FileServiceClient) GetFile(path string, fileRange *FileRange) (*FileStre
 	}
 
 	props, err := getFileProps(resp.headers)
+	if err != nil {
+		return nil, err
```

Comment:
This is one of those mysterious looking errors. If `getFileProps` returns an error and then it is returned by `GetFile`, we'll just get a `ParseUint` error. We won't really know what the problem is unless we dive into the implementation.